config.json

```diff
@@ -1402,6 +1402,19 @@
           "unsafe"
         ]
       },
+      {
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "51c31e6a-b7ec-469d-8a28-dd821fd857d2",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
+        "topics": [
+          "enums",
```

Comment:
As `enums` is a concept (as defined in the `concepts` section of this file), this should go in the `practices` array instead of `topics`. We have [lots of cleanup](https://github.com/exercism/rust/issues/1407) to do in this repo for that, so let's avoid exacerbating the problem.